namespaces/live-1.cloud-platform.service.justice.gov.uk/umar-dev/01-rbac.yaml

```diff
@@ -0,0 +1,13 @@
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: umar-dev-admin
+  namespace: umar-dev
+subjects:
+  - kind: Group
+    name: "github:webops"
```

Comment:
your github team name here, otherwise you won't be able to access the namespace

Reply:
Thank you so much for carefully checking that, have updated the team name to `github:form-builder`